app/assets/stylesheets/ama_layout/layout_components/forms.scss

```diff
@@ -130,31 +141,65 @@ input[type="radio"]:checked + label{
   }
 }
 
+input[type="checkbox"] + label > span:first-of-type{
+  border-radius: 0;
+  max-width: 10%;
+  margin: $base-margin $base-margin*2 $base-margin 0;
+}
+
+.checkbox-label{
+  display: inline-block;
+  width: 80%;
+}
+
 input[type="checkbox"]:checked + label{
   position: relative;
-}
 
-[type="checkbox"]:checked + label:before{
-  content: '✔';
-  background-color: $secondary-color;
-  color: $white;
-  position: absolute;
-  width: 30px;
-  min-width: 30px;
-  max-width: 10%;
-  height: 30px;
-  top: 15px;
-  padding-left: 6px;
-  font-size: $base-font-size*1.3;
-  -moz-border-radius: 50%;
-  border-radius:  50%;
-  margin: 0 $base-margin*2 $base-margin 0;
+  &:before{
+    content: '';
+    display: block;
+    position: absolute;
+    left: 14px;
+    top: 28px;
+    width: 8px;
+    height: 16px;
+    border-right: 3px solid $white;
+    border-top: 3px solid $white;
+    transform:(scaleX(-1) rotate(180deg + -45deg));
+    transform-origin:left top;
+    -webkit-transform:(scaleX(-1) rotate(180deg + -45deg));
+    -webkit-transform-origin:left top;
+    -moz-transform:(scaleX(-1) rotate(180deg + -45deg));
+    -moz-transform-origin:left top;
+    animation:checkbox 0.8s;
+  }
+
+  span:first-of-type{
+    background: $brand-blue-light;
+  }
 }
 
 [type="checkbox"]:checked + label:after{
   margin-left: $base-margin*6;
 }
 
+@keyframes checkbox{
+  0%{
+   height: 0;
+   width: 0;
+   }
+
+  25%{
+    height: 0px;
+    width: 8px;
+  }
+
+  50%{
+    height: 17px;
+    width: 8px;
```

Comment:
I like to define both if they are the only things changing so it's easier to see the changes to both of them. I think some browsers also render these differently and need both defined.